arch/sim/src/sim/up_simsmp.c

```diff
@@ -324,11 +344,22 @@ int up_cpu_start(int cpu)
   /* Initialize the CPU info */
 
   cpuinfo.cpu = cpu;
-  ret = sem_init(&cpuinfo.done, 0, 0);
+
+#ifdef CONFIG_HOST_MACOS
+  done_sema = sem_open(g_sema_name_done, O_CREAT, 0644, 0);
```

Comment:
The name need different for each cpu and for multiple nuttx instance

Reply:
It's a good catch, thanks !